grr/server/grr_response_server/output_plugins/elasticsearch_plugin_test.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+# Lint as: python3
+# -*- encoding: utf-8 -*-
+"""Tests for Elasticsearch output plugin."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+from unittest import mock
+
+from absl import app
+import requests
+
+from grr_response_core.lib import rdfvalue
+from grr_response_core.lib.rdfvalues import client as rdf_client
+from grr_response_core.lib.rdfvalues import client_fs as rdf_client_fs
+from grr_response_core.lib.rdfvalues import flows as rdf_flows
+from grr_response_core.lib.rdfvalues import paths as rdf_paths
+from grr_response_core.lib.util.compat import json
+from grr_response_server import data_store
+from grr_response_server.output_plugins import elasticsearch_plugin
+from grr_response_server.rdfvalues import flow_objects as rdf_flow_objects
+from grr.test_lib import flow_test_lib
+from grr.test_lib import test_lib
+
+KWARGS = 1
```

Comment:
This warrants a comment.

Reply:
Definitely right - I don't even remember what this is for any more ):